temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
+print("hello")
```

Comment:
Print statements should generally be avoided in production code. Consider using logging for better control over output and log levels.
```
import logging

logging.basicConfig(level=logging.INFO)
logging.info('hello')
```